api/ServiceWorkerRegistration.json

```
@@ -514,12 +514,12 @@
               "version_added": false
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "80"
             }
           },
           "status": {
             "experimental": true,
-            "standard_track": false,
+            "standard_track": true,
```

I dug into this a bit. This feature has had a complicated specification history (see also #3530). It doesn't look like there's a spec with `periodicSync` and it's not clear what spec it may ultimately make its way into (e.g., see https://github.com/WICG/BackgroundSync/issues/156). Until there's something a bit more definitive, I'd like to hold off marking this as `true`.

```suggestion
            "standard_track": false,
```

---

Hello Daniel! It's my pleasure to send my first PR to BCD :)
I based my decision on `standard_track` value on 3 points:
- Explanation of this property https://github.com/mdn/browser-compat-data/issues/4650#issuecomment-523094023
- Spec https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html
- This spec is implemented in Chromium-based browsers and shipped in the stable channel

For me, it looked like the case number 2 from the `standard_track` description. On the other hand, this spec is still a "_Draft Community Group Report_". So I agree with you - let's wait for some extra signals to call it a part of the standard.

---

Yes, I admit there's a bit of a judgement call on the standards question. Our [schema docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#status-information) officially say the value should be `true` if the feature is "part of an active […] specification process". But we could probably use a clearer definition of "active." 😄